workspace/libs/shared/utils/src/lib/utils/string-utils.class.ts

```diff
@@ -3,8 +3,10 @@ export class StringUtils {
    * @description
    * Capitalize the first letter of a string.
    */
-  public static capitalize(value: string): string {
-    return (value && typeof value === 'string')
+  public static capitalize(
```

Comment:
Does it make sense to update the code to allow null and undefined instead of updating the tests to not check for null/undefined?  Then you would want to find out where (likely a pipe) it is being used that needs this check and update that instead to perform the null/undefined check.